resources/views/hr/application/index.blade.php

```diff
@@ -46,167 +45,157 @@
+                    <h5 class="open" id="text2">Open</h5>
                 </a>
             </li>
-
-            <li class="nav-item">
-                <a class="nav-item nav-link d-flex align-items-center {{ $status === config('constants.hr.status.in-progress.label') ? 'active bg-info text-white' : 'text-info' }}"
-                    href=/{{ Request::path() }}?status={{ config('constants.hr.status.in-progress.label') }}{{$query_filters}}&round=Trial-Program>
-                    <i class="fa fa-clipboard"></i>&nbsp;
-                    Trial Program
-                    <span
-                        class="ml-1 d-inline-block px-2 py-0 {{ request()->get('round')=='Trial-Program' ? 'active bg-white text-info' : 'bg-info text-white' }}"
-                        style="border-radius: 20px;font-size: 12px;font-weight: 700;">
+            <li id="list-styling">
+                <a  class="btn" id="job-application-listings"
+                    href=/{{ Request::path() }}?status={{ config('constants.hr.status.in-progress.label') }}{{$query_filters}}&round=Trial-Program data-spy="affix" data-offset-top="197">
+                    <sup class = "head1">
                         {{$trialProgramCount}}
-                    </span>
+                    </sup>
+                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"  fill="currentColor" class="bi bi-code trial"  viewBox="0 0 16 16"><path d="M5.854 4.854a.5.5 0 1 0-.708-.708l-3.5 3.5a.5.5 0 0 0 0 .708l3.5 3.5a.5.5 0 0 0 .708-.708L2.707 8l3.147-3.146zm4.292 0a.5.5 0 0 1 .708-.708l3.5 3.5a.5.5 0 0 1 0 .708l-3.5 3.5a.5.5 0 0 1-.708-.708L13.293 8l-3.147-3.146z"/></svg>
+                    <h5 class="trial-program" id="text1">Trial Program</h5>
+                    {{ request()->get('round')=='Trial-Program' ? '' : '' }}
                 </a>
             </li>
-
-            <li class="nav-item">
-                <a class="nav-item nav-link d-flex align-items-center {{ $status === config('constants.hr.status.on-hold.label') ? 'active bg-info text-white' : 'text-info' }}"
+            <li id="list-styling">
+                <a class="btn" id="job-application-listings"
                     href=/{{Request::path() .'?status='. config('constants.hr.status.on-hold.label')}}{{$query_filters}}>
-                    <i class="fa fa-file-text-o"></i>&nbsp;
-                    {{ config('constants.hr.status.on-hold.title') }}
-                    <span
-                        class="ml-1 d-inline-block px-2 py-0 {{ $status === config('constants.hr.status.on-hold.label') ? 'active bg-white text-info' : 'bg-info text-white' }}"
-                        style="border-radius: 20px;font-size: 12px;font-weight: 700;">
+                    <sup class ="head1">
```

Comment:
Use a meaningful name at all places required.
```suggestion
                    <sup class ="application-menu-options-title">
```